### PR TITLE
Add more about storage recommendations for master nodes.

### PR DESCRIPTION
**Pull request by lsgunnlsgunn**
## Description

https://jira.mesosphere.com/browse/DCOS-49954
https://jira.mesosphere.com/browse/DCOS-21142

## Urgency
- [ ] Blocker 
- [ ] High
- [X] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

**Comment by lsgunnlsgunn** [comment]
This PR is for the additional master node system requirements requested in the comments for devices and potential i/o latency issues

**Comment by lsgunnlsgunn** [comment]
Any comments or changes before this gets merged to 1.13?

File: pages/1.13/installing/production/system-requirements/index.md [hunk 1]
```diff
@@ -46,13 +46,16 @@ Workloads that are expected to be continuously available or considered business
...
+
+- RAID controllers with a battery backup unit (BBU).
+- RAID controller cache configured in writeback mode.
 - If separation of storage mount points is possible, the following storage mount points are recommended on the master node. These recommendations will optimize the performance of a busy DC/OS cluster by isolating the I/O of various services.
   | Directory Path | Description |
   |:-------------- | :---------- |
-  | _/var/lib/dcos_ | A majority of the I/O on the master nodes will occur within this directory structure. If you are planning a cluster with hundreds of nodes or intend to have a high rate of deploying and deleting workloads, isolating this directory to dedicated SSD storage is recommended. |
+  | _/var/lib/dcos_ | A majority of the I/O on the master nodes will occur within this directory structure. If you are planning a cluster with hundreds of nodes or intend to have a high rate of deploying and deleting workloads, isolating this directory to dedicated SSD storage on a separate partition is recommended. |
```
Comment (jongiddy) [inline pages/1.13/installing/production/system-requirements/index.md]: The recommendation is to put it "on a separate device", not just a separate partition.

Comment (lsgunnlsgunn) [reply pages/1.13/installing/production/system-requirements/index.md]: Thanks, fixed.

**Pull request merged**